packages/components/src/autocomplete/README.md

```diff
@@ -76,13 +76,6 @@ There are currently two supported actions:
 * "insert-at-caret" - Insert the `value` into the text (the default completion action).
 * "replace" - Replace the current block with the block specified in the `value` property.
 
-#### allowNode
```

Comment:
Should these be removed as well?

https://github.com/WordPress/gutenberg/blob/a195fadc6305aeb78ddc25c64d51e8b72fcbe291/packages/components/src/autocomplete/index.js#L63-L68

https://github.com/WordPress/gutenberg/blob/a195fadc6305aeb78ddc25c64d51e8b72fcbe291/packages/components/src/autocomplete/index.js#L108

Reply:
Yep, good catch.